lib/common/components/Loading.js

```diff
@@ -5,20 +5,27 @@ import React, {Component} from 'react'
 import { Row, Col } from 'react-bootstrap'
 
 type Props = {
+  small?: boolean,
   style?: {[string]: string | number}
 }
 
 export default class Loading extends Component<Props> {
   render () {
-    const {style} = this.props
-    return (
-      <Row>
+    const {small, style} = this.props
+    const icon = (
+      <Icon
+        style={style}
+        className={`fa-spin ${small ? '' : 'fa-5x'}`}
+        type='refresh' />
+    )
+    return small
```

Comment:
What does `small` do here? Could use a comment.

Reply:
The `small` prop is being used both to make the icon smaller by excluding the `fa-5x` class and is also being used to return the icon inline (e.g., directly next to some other text) rather than on its own row. Do you think this should be a new prop called `inline` that is specified separately from `small`?

Reply:
Yeah, maybe call it inline. Whatever the case, please do add a comment.